crates/eframe/src/web/web_runner.rs

```diff
@@ -24,6 +27,9 @@ pub struct WebRunner {
     /// They have to be in a separate `Rc` so that we don't need to pass them to
     /// the panic handler, since they aren't `Send`.
     events_to_unsubscribe: Rc<RefCell<Vec<EventToUnsubscribe>>>,
+
+    /// Used in `destroy` to cancel a pending frame.
+    raf_id: Cell<Option<i32>>,
```

Comment:
let's not involve the royal air force in this

```suggestion
    request_animation_frame_id: Cell<Option<i32>>,
```